app/views/rides/_share_ride.html.erb

```diff
@@ -15,8 +15,9 @@
         data-layout="button" data-size="large">
       <a target="_blank"
          href="https://www.facebook.com/sharer/sharer.php?u=<%= url_encode share_ride_url(ride) %>&amp;src=sdkpreparse"
-         class="fb-xfbml-parse-ignore">
-        Share
+         class="btn waves-effect" style="background: #1877f2">
+         <i class="large material-icons">thumb_up</i>
```

Comment:
Can we put facebook branding here instead?  I want it to be clear we are sharing to facebook